components/colorwheel/index.css

```diff
@@ -11,32 +11,33 @@
  * governing permissions and limitations under the License.
  */
 
-/* Windows High Contrast Mode */
-@media (forced-colors: active) {
-	.spectrum-ColorWheel {
-		--highcontrast-colorwheel-border-color-disabled: GrayText;
-		--highcontrast-colorwheel-fill-color-disabled: Canvas;
+.spectrum-ColorWheel {
+	--spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
```

Comment:
Totally non-blocking, but what do you think about renaming this from `--spectrum-colorwheel-min-width` to `--spectrum-colorwheel-min-inline-size`? I've seen that in a few other places, but I'm not sure if the more "logical" naming is a team preference. Picker has some variables named with the logical naming. 🤷‍♀️ Just a thought. I guess if we rename this one variable, maybe it makes sense to rename the others, too.